src/dmd/mtype.d

```diff
@@ -4258,6 +4258,16 @@ extern (C++) final class TypeFunction : TypeNext
 
     override Type syntaxCopy()
     {
+        import core.memory : GC;
+        import dmd.root.rmem : mem;
+
+        if (mem.isGCEnabled)
+        {
+            GC.collect();
```

Comment:
maybe collect was too much here, it's running since 40 mins now.

Reply:
Interesting. `GC.disable` didn't change https://github.com/dlang/dmd/pull/11593#issuecomment-676339595, but `GC.collect` did. Propbably because the GC will become active if there is not enough memory anyway.